src/include/com/amazonaws/kinesis/video/webrtcclient/NullableDefs.h

```diff
@@ -0,0 +1,94 @@
+/*
+ * Header file to define nullable values for common data types
+ */
+
+#ifndef __KINESIS_VIDEO_WEBRTC_CLIENT_NULLABLEDEFS__
+#define __KINESIS_VIDEO_WEBRTC_CLIENT_NULLABLEDEFS__
+
+#pragma once
+
+#ifdef  __cplusplus
+extern "C" {
+#endif
+
+////////////////////////////////////////////////////
+// Public headers
+////////////////////////////////////////////////////
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wgnu-zero-variadic-macro-arguments"
+#pragma clang diagnostic pop
+
+
+
+#define SET_DEFAULT_AS_NULL(a) \
+    do { \
+        a.isNullable = TRUE; \
+        a.value = 0; \
```

Comment:
This might fail/generate warnings on some compilers. Do we even care about setting the value?

Reply:
Added for sake of completeness. Removed it.